tools/install_demo_configuration.sh

```diff
@@ -373,7 +373,7 @@ echo "plugins.security.enable_snapshot_restore_privilege: true" | $SUDO_CMD tee
 echo "plugins.security.check_snapshot_restore_write_privileges: true" | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
 echo 'plugins.security.restapi.roles_enabled: ["all_access", "security_rest_api_access"]' | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
 echo 'plugins.security.system_indices.enabled: true' | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
-echo 'plugins.security.system_indices.indices: [".opendistro-alerting-config", ".opendistro-alerting-alert*", ".opendistro-anomaly-results*", ".opendistro-anomaly-detector*", ".opendistro-anomaly-checkpoints", ".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opendistro-notifications-*", ".opendistro-notebooks", ".opendistro-asynchronous-search-response*"]' | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
+echo 'plugins.security.system_indices.indices: [".os_ml_model_result", ".opendistro-alerting-config", ".opendistro-alerting-alert*", ".opendistro-anomaly-results*", ".opendistro-anomaly-detector*", ".opendistro-anomaly-checkpoints", ".opendistro-anomaly-detection-state", ".opendistro-reports-*", ".opendistro-notifications-*", ".opendistro-notebooks", ".opendistro-asynchronous-search-response*"]' | $SUDO_CMD tee -a "$OPENSEARCH_CONF_FILE" > /dev/null
```

Comment:
Alerting plugin plans to move their plugin index out of system indices https://github.com/opensearch-project/alerting/issues/125. Should we put new plugin index into system indices list? cc @skkosuri-amzn

Reply:
@spbjss Did you get chance to close this?

Reply:
https://github.com/opensearch-project/alerting/issues/125#issuecomment-880214156  based on this discussion,  .os_ml_model_result should be part of system index. 

Considering .os_ml_model_result --> stores model which will be used by ML plugin. Any change in models stored in this index must be allowed only via APIs exposed by ML plugin.  So, we can avoid accidental update to this index via any users.

Reply:
Updated, thank you!

Reply:
Updated the ml index name to '.plugins-ml-model' based on the discussion with @dblock @ylwu-amzn  and @saratvemulapalli